src/binding-language.js

```diff
@@ -79,19 +79,17 @@ export class TemplatingBindingLanguage extends BindingLanguage {
   }
 
   parseContent(resources, attrName, attrValue){
-    var parts = attrValue.split(this.interpolationRegex), i, ii;
-    if (parts.length <= 1) { //no expression found
+    var parts = [];
+    parse(attrValue, (type, start, end, extra) => {
+      var value = attrValue.substring(start, end) + (extra || '');
+      var expr = type === 'expr' ? this.parser.parse(value) : null;
+      parts.push({type, value, expr});
+    });
+
+    if (exprs.length == 0) { //no expression found
```

Comment:
@Alxandr -

@EisenbergEffect asked that we compare the two PRs and determine the perf/memory pressure.  I'm getting an error running my test-suite on this line: "exprs not defined".  Would you mind copying [this piece of code](https://github.com/jdanyow/templating-binding/blob/aa416e1bfc6767e3aec3670c2579f3f0970b4ad2/test/binding-language.spec.js#L105-L156) into your copy and making sure all the tests pass, then update this PR so we can continue performance tuning?

Thanks!
-jd